src/block/BlockToolType.php

```diff
@@ -40,5 +40,6 @@ private function __construct(){
 	public const AXE = 1 << 3;
 	public const SHEARS = 1 << 4;
 	public const HOE = 1 << 5;
+	public const MACE = 1 << 6;
```

Comment:
The tool type only must be added here for variants like diamond and iron versions of the item.